__tests__/app.test.js

```diff
@@ -81,11 +82,38 @@ describe('GET/api/reviews', () => {
     });
     test('date is ordered in desc order', () => {
         return request(app).get('/api/reviews').then(({body}) => {
-            let firstReview = body[0]
-            let lastReview = body[body.length - 1]
+            expect(body).toBeSortedBy('created_at', {descending: true})
+        })
+    });
+});
 
-            expect(firstReview.created_at).toEqual('2021-01-25T11:16:54.963Z')
-            expect(lastReview.created_at).toEqual('1970-01-10T02:08:38.400Z')
+describe('GET/api/reviews:review_id', () => {
+    test('should return a status code of 200', () => {
+        return request(app).get('/api/reviews/1').expect(200)
+    });
+    test('should contain the following keys', () => {
+        return request(app).get('/api/reviews/1').then(({body}) => {
+            expect(body).toHaveProperty("review_id", expect.any(Number))
```

Comment:
Have a look at `.objectContaining( )` and see if it's your thing in circumstances like this.